packages/documentation/docs/playground/overview.md

```diff
@@ -0,0 +1,176 @@
+# Overview
```

Comment:
Taken from the `localenv` README. Maybe we just change that README to just link to this doc? So they don't ever diverge

Reply:
My gut instinct is to have the setup instructions in the repo but I guess if it's linked then I would have found it in the same way (visit repo -> read readme -> proceed to localenv link). I agree we shouldn't really have multiple versions of the same thing. 

I was thinking about this with the README.md and docusaurus's docs/introduction/overview but that's a bit different because we really need to answer "What is rafiki" at the beginning of the readme (as we currently do).

I do feel like the setup instructions are a bit buried in the `playground/overview` document but if we link specifically to the running-the-local-environment section that's probably not an issue.